### DISCUSSION TITLE
How do I stop certain elements from being parsed?

### DISCUSSION THREAD

**Discussion by tyteen4a03** [remarkjs/remark#1220] [Q&A]
In Remark 13+, how do I stop certain Markdown elements (e.g. italics, blockquotes) from being parsed? They should not render into HTML elements, but rather kept as is in the output.

https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-disable-tokenizers doesn't seem to support Remark 13+.

**Comment by ChristianMurphy** [comment]
Welcome @tyteen4a03! 👋 
It is possible to disable certain markdown constructs.
You can find context in the previous thread where this was asked https://github.com/orgs/remarkjs/discussions/757 and an example in the micromark documentation https://github.com/micromark/micromark#case-turn-off-constructs (micromark is the lower level parser/tokenizer inside remark)
Many of the `zmarkdown` plugins are broken currently in remark 13+, if you prefer to leverage those, feel free to contribute to their upgrade efforts https://github.com/zestedesavoir/zmarkdown/issues/416
